paint/PAINT_Shape.h

```diff
@@ -13,5 +23,8 @@ namespace paint
 		Shape(Shape && that) = default;
 		Shape& operator=(const Shape & that) = default;
 		Shape& operator=(Shape && that) = default;
+
+		virtual std::vector<gfx::Line> shapeGeometry(win::Coords startCoords, win::Coords currentCoords) = 0;
```

Comment:
Rename to `getGeometry()`. Compare how it reads as English and the information conveyed:

```
auto g = shape->shapeGeometry(start, pos) ;
auto g = shape->getGeometry(start, pos);
```
**SHOULD**

Reply:
done